### ISSUE TITLE
Feature Request: optionally use blessed.Image element for image rendering.

### ISSUE THREAD

**Issue by chjj** [mkaminsky11/imgurize#1]
If w3m is installed, blessed.Image will call w3mimgdisplay to draw the actual image over the terminal window. This probably only works for X11 terminals though (xterm, urxvt, gnome-terminal/vte, etc).

I like this project. Good work.

**Comment by mkaminsky11** [comment]
Yeah, I thought about that, but for the sake of compatibility (and because ascii art is awesome), I decided against it

**Comment by mkaminsky11** [comment]
but i'll make it an option

**Comment by chjj** [comment]
Yeah, I understand. I just thought it might work well as an option. Plus I want to see people make more use of the Image element. :)
